lib/dockerize_rails/docker_commands/helpers.rb

```diff
@@ -0,0 +1,10 @@
+module DockerizeRails
+  module DockerCommands
+    module Helpers
+      def self.get_services_from_docker_compose
```

Comment:
__[rubocop]__
* Do not prefix reader method names with `get_`. [:ref](https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names)

<!-- Comment by [SideCI](https://sideci.com) -->